config/application.rb

```diff
@@ -24,7 +24,7 @@
 module Psulcat
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults '6.0'
+    config.load_defaults 6.0
```

Comment:
Nice catch. Does this actually change behavior or is it just "more correct"?

Reply:
I think it is correct either way, load_defaults does `to_s` to it anyway, I ran `bin/rails app:update` to see what might have been changed, just got this changed during that, thought it looks cleaner nothing else I think.